api/app/src/command/medical/patient/get-patient.ts

```diff
@@ -19,6 +21,35 @@ export const getPatients = async ({
   return patients;
 };
 
+export const getPatientByDemo = async ({
+  facilityId,
+  cxId,
+  demo,
+}: {
+  facilityId: string;
+  cxId: string;
+  demo: PatientData;
+}): Promise<Patient | null> => {
+  const patient = await PatientModel.findOne({
+    where: {
+      cxId,
+      facilityIds: [facilityId],
+      data: {
+        firstName: demo.firstName,
+        lastName: demo.lastName,
+        personalIdentifiers: {
+          [Op.in]: demo.personalIdentifiers,
+        },
```

Comment:
will this work if the `demo.personalIdentifiers` is null? or vice versa?

Reply:
personalIdentifiers cant be null but it could be empty